README.md

```diff
@@ -63,12 +63,13 @@ yarn global add nx
 ```bash
 git clone https://github.com/ethereum/remix-project.git
 ```
-* Build `remix-project`:
-```bash
-cd remix-project
-yarn
-yarn serve
-```
+* Build and Run `remix-project`:
+
+1. Move to project directory: `cd remix-project`
+2. Install dependencies: `yarn install` or simply run `yarn`
+3. Build Remix libraries: `yarn run build:libs`
```

Comment:
yarn run build:libs isn't required any more. Like @bunsenstraat said, its done when you "yarn serve" libs is like the ui lib. Maybe this might be required when you want to publish libs but not for just normal dev flow